web/src/components/ui/AlertDialog.test.tsx

```diff
@@ -1,14 +1,15 @@
-import React from 'react'
```

Comment:
Try to avoid changing unrelated files.

Reply:
I got a couple of alert in Intellj complaining about the use of " and '